.github/workflows/ci.yml

```diff
@@ -129,7 +129,12 @@ jobs:
         run: pnpm build
       - if: github.event_name == 'pull_request'
         run: pnpm affected:build -- --base=$BASE_BRANCH
-      - uses: actions/upload-artifact@v2
+      - name: 'Spectacular: Remove @angular/forms dependency'
+        run:
+          pnpx json -I -f dist/packages/spectacular/package.json -e "delete
+          this.dependencies['@angular/forms'];" || true
```

Comment:
Why would the @angular/forms dependency be in the package.json if originally is not there?

Reply:
`ng-packagr` detects a single use of `@angular/forms`. in the tour of heroes crisis center library. Spectacular has one test which imports from that library. Apparently, it's not able to differentiate between test dependencies and runtime dependencies.